odoo/fields.py

```diff
@@ -1865,6 +1866,11 @@ def write(self, records, value):
         cache = records.env.cache
         cache_value = self.convert_to_cache(value, records)
         records = cache.get_records_different_from(records, self, cache_value)
+        if records and cache_value is None:
```

Comment:
BEHAVIOUR: this is not yet correct, we want different behaviour when writing "" or False
Note that `update_field_translations` calls write with the value that is read!